src/crab/bitset_domain.cpp

```
@@ -6,23 +6,23 @@
-        if (b.non_numerical_bytes[EBPF_STACK_SIZE + i]) {
+    for (int i = -EBPF_TOTAL_STACK_SIZE; i < 0; i++) {
+        if (b.non_numerical_bytes[EBPF_TOTAL_STACK_SIZE + i]) {
             continue;
         }
         if (!first) {
             o << ", ";
         }
         first = false;
-        o << "[" << EBPF_STACK_SIZE + i;
+        o << "[" << EBPF_TOTAL_STACK_SIZE + i;
         int j = i + 1;
```

cppcheck

[performance] Function parameter 'non_numerical_bytes' should be passed by const reference.

(passedByValue)